<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Omni</class>
 <widget class="QWidget" name="Omni">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>103</width>
     <height>172</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="xLabel">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="yLabel">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="zLabel">
        <property name="text">
         <string>Z:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rollLabel">
        <property name="text">
         <string>Roll:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pitchLabel">
        <property name="text">
         <string>Pitch:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="yawLabel">
        <property name="text">
         <string>Yaw:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLCDNumber" name="xLCD"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="yLCD"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="zLCD"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="rollLCD"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="pitchLCD"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="yawLCD"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="outputText">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="sphereSizeLabel">
      <property name="text">
       <string>Sphere Size</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sphSizeSpinBox">
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>80.000000000000000</double>
      </property>
      <property name="value">
       <double>40.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
